WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -231,6 +231,10 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 
 				registeredAliceClientAndCircuits = await ProceedWithInputRegAndConfirmAsync(smartCoins, roundState, linkedCts.Token).ConfigureAwait(false);
 			}
+			catch(OperationCanceledException ex)
+			{
+				return new CoinJoinResult(false);
```

Comment:
What happens if this is not here?

Reply:
This is changing the behavior. We are waiting for that exception here: 
https://github.com/molnard/WalletWasabi/blob/master/WalletWasabi/WabiSabi/Client/CoinJoinManager.cs#L396